docs/components/servo/pi.md

```diff
@@ -86,11 +86,11 @@ The following attributes are available for `pi` servos:
 | ---- | ---- | --------- | ----------- |
 | `pin` | string | **Required** | The {{< glossary_tooltip term_id="pin-number" text="pin number" >}} of the pin the servo's control wire is wired to on the [board](/components/board/). |
 | `board` | string | **Required** | `name` of the [board](/components/board/) the servo is wired to. |
-| `min` | float | Optional | The minimum angle in degrees that the servo can reach. <br> Default = `0.0` <br> Range = [`0.0`, `180.0`] |
-| `max` | float | Optional | The maximum angle in degrees that the servo can reach. <br> Default = `180.0` <br> Range = [`0.0`, `180.0`] |
+| `min` | float | Optional | The minimum angle in degrees you want your servo to reach while rotating. <br> Default = `0.0` <br> Range = [`0.0`, `180.0`] |
+| `max` | float | Optional | The maximum angle in degrees you want your servo to reach while rotating. <br> Default = `180.0` <br> Range = [`0.0`, `180.0`] |
 | `starting_position_degs` | float | Optional | Starting position of the servo in degrees. <br> Default = `0.0` <br> Range = [`0.0`, `180.0`] |
 | `hold_position` | boolean | Optional | If `false`, power down a servo if it has tried and failed to go to a position for a duration of 500 milliseconds. <br> Default = `true` |
-| `max_rotation_deg` | int | Optional | The maximum angle the servo can rotate. Must be in between `min` and `max`. <br> Default = `180` |
+| `max_rotation_deg` | int | Optional | The maximum angle that you know your servo can possibly rotate to, according to its hardware. Refer to your servo's data sheet for clarification. Must be in between the values you choose for `min` and `max`. <br> Default = `180` |
```

Comment:
I wonder whether the "Must be in between the values you choose for min and max" is actually accurate. I know it was here before but that part doesn't make sense to me, since min and max could be inside of that range.
Based on https://github.com/viamrobotics/rdk/blob/main/components/board/pi/impl/servo.go I think we can say:
```suggestion
| `max_rotation_deg` | int | Optional | The maximum angle that you know your servo can possibly rotate to, according to its hardware. Refer to your servo's data sheet for clarification. Must be greater than or equal to the value you set for `max`. <br> Default = `180` |
```

Reply:
Ahh, this does seem right. It seems like it's actually checking << `min` and `max` , though, so I will edit to be that